### PR TITLE
doc/ceph-volume: initial docs for zfs/inventory and zfs/api

### PR DESCRIPTION
**Pull request by wjwithagen**
Fixes: https://tracker.ceph.com/issues/42722

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: doc/ceph-volume/zfs/inventory.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.. _ceph-volume-zfs-inventory:
+
+``inventory``
+=============
+The ``inventory`` subcommand queries a host's disc inventory thru GEOM and provides
```
Comment (jan--f) [inline doc/ceph-volume/zfs/inventory.rst]:
```suggestion
The ``inventory`` subcommand queries a host's disc inventory through GEOM and provides
```

**Comment by wjwithagen** [comment]
> 
> 
> I mightbe mistaken here so apologies in advance, but my understanding is that the zfs plugin is specifically targetting Freebsd. If that is correct this limitation should be explicitly mentioned in toe doc pages. Also I'm not sure this module is currently packaged by default (maybe only for Freebsd?).
> 
> Otherwise we might cause confusion as zfs is also available for linux.

I'm working out the packaging on FreeBSD, so you might be right there.
And yes, ZFS now assumes an underlying FreeBSD infra, but it would not be too hard to actually make it work on ZoL as well.

Will try to put an alert in

**Comment by jan--f** [comment]
jenkins test make check

**Comment by wjwithagen** [comment]
jenkins test make check

**Comment by wjwithagen** [comment]
@alfredodeza 
Can you merge this?
thanx

**Pull request merged**